app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,15 @@
+//
+// Variables
+// --------------------------------------------------
+
+
+//== Colors
+//
+//## Gray colors for use across Coursemology.
+
+$gray-base:              #000 !default;
```

Comment:
@wangqiang1208 @fonglh @kxmbrian @jsyeo @weiqingtoh ...american or british spelling? lol... i can't remember what the style guides say